hsmodels/schemas/enums.py

```diff
@@ -87,7 +87,8 @@ class DateType(TermEnum):
     created = str(DCTERMS.created)
     valid = str(DCTERMS.valid)
     available = str(DCTERMS.available)
-    published = str(DCTERMS.published)
+    published = str(HSTERMS.published)
+    review_started = str(HSTERMS.reviewStarted)
```

Comment:
Need tests around reviewStarted

Reply:
Thanks @sblack-usu I modified the schemas and added some basic tests